db/migrate/20220203235909_create_rule_group_rules.rb

```diff
@@ -0,0 +1,9 @@
+class CreateRuleGroupRules < ActiveRecord::Migration[7.0]
+  def change
+    create_table :rule_group_rules, id: :uuid do |t|
+      t.references :rule_group, type: :uuid
+    end
+    add_column :rule_group_rules, :rule_id, :uuid
+    add_index :rule_group_rules, :rule_id, unique: true
```

Comment:
This doesn't really make sense, you can have these columns into the `create_table`. This makes 3 SQL queries instead of one.

Reply:
https://github.com/RedHatInsights/compliance-backend/blob/master/db%2Fmigrate%2F20210201233645_drop_profile_hosts.rb#L13

Reply:
```
t.references :foo, foreign_key: true, index: {unique: true}
```